### PR TITLE
Improve style and schema

### PR DESCRIPTION
**Pull request by nlehuby**
These are the changes made by @djakk and @frodrigo during JungleBus Weekend.

### CODE REVIEW

File: settings/post-process.sql [hunk 1]
```diff
@@ -28,3 +28,70 @@ alter table osm_bus_points add routes_at_stop character varying;
...
+--- Add average stop distance on osm_bus_route_members and osm_bus_points
+CREATE TEMP TABLE tt_osm_bus_route_members_avg_distance AS
+SELECT
+  rel_osm_id,
+  avg(distance) AS avg_distance
+FROM
+(
+SELECT
+  rel_osm_id,
+  ST_DistanceSphere(geometry, lag(geometry, 1, geometry) OVER (PARTITION BY rel_osm_id ORDER BY rel_osm_id, member_index)) AS distance
```
Comment (nlehuby) [inline settings/post-process.sql]: There is something wrong here:
on the ile-de-france pbf, it crashes, with the following error message: `ERROR:  distance returned negative!`

**Comment by nlehuby** [comment]
Sorry for the late review, we have been quite busy these days!

This is a very good increment, I think we can merge this and move on with the TODOLIST.

File: settings/post-process.sql (hunk 1 above)
Comment (frodrigo) [reply settings/post-process.sql]: It's work for me. Do you have more details?
postgis 2.4.3

Comment (frodrigo) [reply settings/post-process.sql]: Got it with Cuba extract.

**Comment by frodrigo** [comment]
Add a commit to fix negative distance.

**Comment by prhod** [comment]
Hi everybody, I merged the PR of @nlehuby on master causing a conflict here. Resolving it will cause conflict in PR https://github.com/Jungle-Bus/VapourTrail/pull/11 . I think we should  let it like this until PR 9 et 11 are merged into WE.
I'll try to take a look at all the PR soon.

**Comment by frodrigo** [comment]
I will prefer if we merge #4 first.
#9 & #11 are next steps, not improvement of #4.
Once #4 is merged I will resolve conflicts and repush #9 & #11.

**Comment by prhod** [comment]
As requested and checked with @nlehuby, I merged this PR. Thanks !

**Pull request merged**